app/src/lib/git/commit.ts

```diff
@@ -94,6 +114,55 @@ export async function createMergeCommit(
   }
 }
 
+async function stageManualConflictResolution(
+  repository: Repository,
+  file: WorkingDirectoryFileChange,
+  manualResolution: ManualConflictResolution
+): Promise<boolean> {
+  const { status } = file
+  // if somehow the file isn't in a conflicted state
+  if (!isConflictedFileStatus(status)) {
+    console.error(`tried to manually resolve unconflicted file (${file.path})`)
```

Comment:
`log.error` to ensure these cases are flushed to our logging infrastructure if they happen in the wild